src/kaleidoscope/driver/BaseKeyScanner.h

```diff
@@ -29,6 +29,8 @@ class BaseKeyScanner {
 
   typedef typename _KeyScannerDescription::KeyAddr KeyAddr;
 
+  static void handleKeyswitchEvent(Key mappedKey, KeyAddr key_addr, uint8_t keyState);
```

Comment:
Moved this to `BaseKeyScanner` to make it work for all derived key scanner classes.